### PR TITLE
Fix redirect file path

### PR DESCRIPTION
**Pull request by akito19**
Now, our redirect settings are breaking...
I used `_redirects` file but the file doesn't fire on redirection.
So I made `netlify.toml` instead of `_redirects`.

It is little wordy...

See: https://www.netlify.com/docs/redirects/

fixup! Fix _redirects file path
` [db54c3c](/sider/sider-docs/pull/99/commits/db54c3c5029c43cfbac44dfcb369191f738006bd) `

### CODE REVIEW

**Comment by ybiquitous** [comment]
Is new `website/build/sider-docs/_redirects` file commited?

**Comment by akito19** [comment]
> No redirect rules processed
> This deploy did not include any redirect rules. Learn more about redirects.

🤔

File: website/_redirects [hunk 1]
```diff
@@ -7,3 +7,4 @@
 /on-premises/quickstart/* /enterprise/quick-start/:splat
 /on-premises/testing  /enterprise/testings/guide
 /on-premises/releases* /enterprise/releases/changelog:splat
+>>>>>>> 30a3115... fixup! Remove unnecessary files
```
Comment (ybiquitous) [inline website/_redirects]: Ooops! 😳

Comment (akito19) [reply website/_redirects]: 🙃

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from = "/on-premises"
+to = "/enterprise/releases/changelog"
+
+[[redirects]]
+from = "/on-premises/quick-start"
+to = "/enterprise/quick-start/setup"
+
+[[redirects]]
+from = "/on-premises/quickstart/*"
+to = "/enterprise/quick-start/:splat"
```
Comment (ybiquitous) [inline netlify.toml]: [question] Does this redirect work?

https://deploy-preview-99--sider-docs.netlify.com/on-premises/quickstart

File: netlify.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+[[redirects]]
+from = "/on-premises/testing"
+to = "/enterprise/testings/guide"
```
Comment (ybiquitous) [inline netlify.toml]:
```suggestion
to = "/enterprise/testing/guide"
```

File: netlify.toml (hunk 2 above)
Comment (ybiquitous) [reply netlify.toml]: Sorry, these work!

- https://deploy-preview-99--sider-docs.netlify.com/on-premises/quickstart/setup
- https://deploy-preview-99--sider-docs.netlify.com/on-premises/quickstart/update

**Pull request merged**